nexus/src/app/instance.rs

```diff
@@ -1833,18 +1834,16 @@ impl super::Nexus {
 /// Invoked by a sled agent to publish an updated runtime state for an
 /// Instance, returning an update saga for that instance (if one must be
 /// executed).
-pub(crate) async fn notify_instance_updated(
+pub(crate) async fn process_vmm_update(
```

Comment:
And, while I'm thanking you for renaming things that I really should've renamed myself, thanks for changing this to something other than `notify_instance_updated_2` :)